lib/is_rateable/acts_as_rateable.rb

```diff
@@ -11,6 +11,9 @@ def acts_as_rateable(options = {})
 
         has_many :ratee_ratings, as: :ratee, class_name: 'IsRateable::Rating'
 
+        def above_average_rating(rating)
```

Comment:
@RustComet It's a class method / scope that returns the collection of objects above the supplied rating, i.e. `Movie.above_average_rating(5)`

`LocalInstanceMethods` is only for instance methods, i.e. `@movie.stuff`.